### ISSUE TITLE
TST: test_coordinate_operation_grids__alternative_grid_name failing with latest PROJ

### ISSUE THREAD

**Issue by snowman2** [pyproj4/pyproj#1078]
https://github.com/pyproj4/pyproj/runs/6398398042

Possibly related: https://github.com/OSGeo/PROJ/pull/3180

**Comment by snowman2** [comment]
git bisect with PROJ:
```
5298fc0bfe3b73a994f0ff668f3392b803af45e5 is the first bad commit
commit 5298fc0bfe3b73a994f0ff668f3392b803af45e5
Author: Even Rouault <even.rouault@spatialys.com>
Date:   Tue May 3 20:46:53 2022 +0200

    proj_coordoperation_get_grid_used(): make it work more reliably when networking is enabled (fixes #3176)

 src/filemanager.cpp      | 59 ++++++++++++++++++++----------------------------
 src/filemanager.hpp      |  6 +++--
 src/iso19111/factory.cpp | 34 ++++++++++++++--------------
 test/unit/test_c_api.cpp | 10 ++++++--
 4 files changed, 54 insertions(+), 55 deletions(-)
```

**Comment by snowman2** [comment]
Need to handle scenario where the physical grid file is found.
